Backend/Model/Entities/Endereco.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Model.Entities
+{
+    public class Endereco
```

Comment:
Adicionar herança de ValueObject. Isso define de forma mais explicita que isso é uma owned entity

Reply:
Não existe IValueObject

Reply:
https://stackoverflow.com/questions/61706738/what-is-owned-entity-when-and-why-to-use-owned-entity-in-entity-framework-core